### ISSUE TITLE
a11y module describes series on each redraw, even if nothing changed

### ISSUE THREAD

**Issue by pawellysy** [highcharts/highcharts#20226]
#### Expected behaviour
The a11y function should fire only, when something changed in the chart.

#### Actual behaviour
`A11y`'s `SeriesDescriber.DescribeSeries` method is fired on each redraw, even though nothing changed.

#### Live demo with steps to reproduce
https://jsfiddle.net/BlackLabel/kw5bpa9x/1/

Visit this demo, open console, and leave logpoint in the `SeriesDescriber.describeSeries`. 
Click anywhere in the chart's plotArea

see, that after every redraw, which is fired on click, the method above is fired. 

#### Affected browser(s)
v11.2

**Comment by AndrzejBuleczka** [comment]
There is closely related issue raised concerning performance, where a11 module affects performance with 1-2 times longer re-renders on zoom and pannning: https://www.highcharts.com/forum/viewtopic.php?t=52709.
